Pinta.Core/Effects/Utility.cs

```diff
@@ -194,14 +188,10 @@ public static int ColorDifferenceSquared (ColorBgra a, ColorBgra b)
 		return diffSq / 3;
 	}
 
-	public static RectangleI[] InflateRectangles (RectangleI[] rects, int len)
+	public static IEnumerable<RectangleI> InflateRectangles (IEnumerable<RectangleI> rects, int len)
```

Comment:
I'm fine to remove this since it's unused. It was added a long time ago and usages of it were removed since

Reply:
Nice, the less unused methods the better. I've removed it